packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts

```diff
@@ -555,7 +555,7 @@ export class EnterForceIdeUri implements ParametersGatherer<ForceIdeUri> {
     }
 
     return null; // all good
-  }
+  };
```

Comment:
See https://github.com/palantir/tslint/issues/3216.

More info below in tslint.json.